core/unix/signal_private.h

```diff
@@ -191,7 +191,7 @@ typedef _STRUCT_UCONTEXT /* == __darwin_ucontext */ kernel_ucontext_t;
  * (these are from /usr/src/linux/arch/i386/kernel/signal.c for kernel 2.4.17)
```

Comment:
it seems like vera++ 1.2.1 does not properly recognize include tokens. I've tried adding

    +    foreach token [getTokens $fname 1 0 -1 -1 {pp_include}] {
    +        set nline [lindex $token 1]
    +        report $fname $nline "Found"
    +    }

but that never failed with `pp_include` ....